features/facebookImport/src/components/buttons/infoButton/infoButton.css

```diff
@@ -1,14 +1,5 @@
 .info-button {
-    width: 64px;
-    height: 48px;
-    border-radius: 25px 0 0 25px;
     position: absolute;
     right: 0;
-    background-color: var(--color-gray-800);
-    filter: drop-shadow(0px 2px 4px rgba(0, 0, 0, 0.06)),
-        drop-shadow(0px 4px 6px rgba(0, 0, 0, 0.1));
 }
 
-.info-button img {
```

Comment:
Again, not entirely clear why you delete this; the commit message mentions that you're using IconButton in InfoButton, but IconButton still uses images. I don't see an equivalent styling for IconButton (I guess it's [this one](https://github.com/polypoly-eu/polyPod/blob/main/feature-utils/poly-look/src/react-components/buttons/iconButton.css). So is this simply an unrelated change, adapting better to the Figma requirements?

Reply:
For me, to call this pop up -> swiper change feature complete, everything (this includes the button that opens the swiper) needs to be aligned with the new Figma requirements. Unfortunately we are not 100% there yet (I didn't do much about the chart images). It's also the reason I've included `and related components` in the PR title and something similar for the JIRA ticket. About the styling, IconButton already takes care of that, so we only need the positioning.